mods/ts/rules/civilian-structures.yaml

```diff
@@ -18,6 +19,18 @@ ABAN01:
 
 ABAN02:
 	Inherits: ^CivBuilding
+	HitShape:
```

Comment:
Eh, no. Forgot to finish this one before pushing, although not using a single 5x3 rectangle is intentional (I wanted to match the art).